apps/studio/prisma/schema.prisma

```diff
@@ -198,6 +201,19 @@ model SiteMember {
   @@id([siteId, userId])
 }
 
+model ResourcePermission {
```

Comment:
this seems to track perms on a resource-level?

for a site-wide perms ie. a user can access all resources in this site as a Admin / Editor / Publisher, how will we represent this?